### ISSUE TITLE
Add QEMU fuzzer

### ISSUE THREAD

**Issue by XVilka** [ispras/oss-sydr-fuzz#107]
Currently it's fuzzed by OSS-fuzz:
- https://www.qemu.org/docs/master/devel/fuzzing.html
- https://github.com/google/oss-fuzz/tree/master/projects/qemu

**Comment by SweetVishnya** [comment]
Yeah, that's a nice idea! Some our student may add it one day. And PRs are always welcomed ;)
